contrib/format-excel/src/test/java/org/apache/drill/exec/store/excel/TestExcelFormat.java

```diff
@@ -382,6 +382,26 @@ public void testExplicitSomeQueryWithCompressedFile() throws Exception {
     new RowSetComparison(expected).verifyAndClearAll(results);
   }
 
+  @Test
+  public void testFileWithDoubleDates() throws Exception {
+    String sql = "SELECT `Close Date`, `Type` FROM dfs.`excel/comps.xlsx` WHERE style='Contemporary'";
```

Comment:
Do we need to add new file rather than updating existing ones? Project size matters...

Reply:
I removed this and incorporated the data into the one test file.  However, the additional file was only 15kb.  I understand that we want to keep Drill as lightweight as possible, but does 15kb really make that much of a difference?

Reply:
If we’ll look at this from the perspective that everybody will start adding files, this will increase project size.
Generally, we should rather than adding new files:
1. reuse existing files;
2. generate files at runtime, for example for excel files this could have been done fairly easy.

Reply:
Ok.  Noted for future.  Thanks!